### ISSUE TITLE
Kernel appears to have died. It will restart automatically.

### ISSUE THREAD

**Issue by theyusifova** [DOsinga/deep_learning_cookbook#91]
I have dataset, about 40 million row, when i use great_expectations with query, it show me, The kernel for /gx_final_ts(temp_query)/great_expectations/uncommitted/edit_ts_s.ipynb appears to have died. It will restart automatically.

**Comment by DOsinga** [comment]
are you running this locally? what does it say in terminal where you started the notebook? Given that you have a large dataset, I could imagine it ran out of memory

**Comment by theyusifova** [comment]
Yes it running in locally,when i use great_expectations with query in show
me kernel died, but without query it running without problems

19 dek 2023, Ç.a. tarixində 16:32 tarixində Douwe Osinga <
***@***.***> yazdı:
